tests/test_fileIO.py

```diff
@@ -27,4 +28,27 @@ def test_read_dynamx(self):
         assert data['end'][0] == 17
 
         with pytest.raises(ValueError):
-            data = read_dynamx(self.fpath, intervals=('foo', 'bar'))
+            data = read_dynamx(self.fpath, intervals=('foo', 'inclusive'))
+        with pytest.raises(ValueError):
+            data = read_dynamx(self.fpath, intervals=('inclusive', 'bar'))
+
+        with open(self.fpath, mode='r') as f:
+            read_dynamx(StringIO(f.read()))
+
+
+    def test_methods(self):
+        data = read_dynamx(self.fpath)
+        path = directory / 'test_data' / 'ecSecB_guess.txt';
+
+        #testing fmt_export
+        with pytest.raises(ValueError):
+            fmt_export(data, delimiter=',', width="foo")
+
+        #testing txt_to_protein
+        txt_to_protein(path)
+
+        #testing txt_to_np
+        with open(path, mode='r') as f:
+            txt_to_np(StringIO('f'))
```

Comment:
Same for this function

`array = txt_to_np(...)`
then 
assert array[some selection] == known_value